app/assets/javascripts/discourse/app/templates/components/search-result-entries.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="fps-result-entries">
+  {{#each posts as |result|}}
+    {{search-result-entry result=result}}
```

Comment:
is this ever used to display something else than a post ?

If no, I think we should just try to use the post wording: `{{search-result-post post=post}}`

Reply:
yes if it's used elsewhere just keep it :+1:

Reply:
`post` it is, changes done